sql-plugin/src/main/scala/com/nvidia/spark/rapids/GpuSortExec.scala

```diff
@@ -296,13 +295,13 @@ case class GpuOutOfCoreSortIterator(
         sorted.add(sp)
       }
     } else {
-      val splitIndexes = if (sortedOffset >= 0) {
+      val splitIndexes = if (sortedOffset > 0) {
```

Comment:
nit, since we use `sortedOffset > 0` in three different places, maybe it would be good to create a `val` above this line and use that? 

```
val offsetIsNonZero = sortedOffset > 0
```